.github/workflows/sam-automation-tests.yaml

```diff
@@ -42,7 +53,7 @@ jobs:
           repo: broadinstitute/terra-github-workflows
           ref: refs/heads/main
           token: ${{ secrets.BROADBOT_TOKEN}} # github token for access to kick off a job in the private repo
-          inputs: '{ "repository": "${{ github.event.repository.full_name }}", "ref": "${{ steps.extract-branch.outputs.ref }}" }'
+          inputs: '{ "repository": "${{ github.event.repository.full_name }}", "ref": "${{ steps.extract-branch.outputs.ref }}", "sam-release-tag": "${{ steps.tag.outputs.tag }}" }'
```

Comment:
I'm assuming that this workflow builds AND releases Sam?  My initial reaction is that these should be 2 separate steps.  But if they're one step, that seems fine too I guess, I don't know.  If it's easy to separate them, I guess I would.  If it's a pain, then don't worry about it.

Reply:
Great suggestion! We can make build and release as separate steps within the same job. Will create a separate ticket for this.